examples/http_config.json

```diff
@@ -0,0 +1,38 @@
+{
+    "server": {
+        "port": 8849,
+        "read_timeout": 5,
+        "write_timeout": 30
+    },
+    "statsd": {
+      "enabled": false
+    },
+    "sources": {
+        "default": {
+            "type": "http"
```

Comment:
Nevermind—this is the source type. I was thinking of [@eager's comment](https://github.com/eager/halfshell/pull/1/files#r30765687).